### ISSUE TITLE
FAQ "contact" section should have logical headings and be more easily discoverable

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#16]
# FAQ "contact" section should have logical headings and be more easily discoverable [ ⚠️ Should ]

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description
The contact section currently has the following issues:

- Contact is a `h3`, which accidentally places it as a child to one of the FAQ headings
- Accessibility and general questions are not headings (despite appearing as headings), making them harder to discover by screen reader users
- The contact section will be difficult to discover for screen reader users as:
  - it's located deep near the bottom of the DOM
  - there's a large volume of h2s above it

<img width="1486" alt="Screen Shot 2022-05-01 at 2 32 05 PM" src="https://user-images.githubusercontent.com/14154792/166159661-bda3b0c6-5cf1-44a4-8d99-a9f213c46c86.png">

## Recommendation or success criteria
- [x] Contact is a `h2`, not a `h3`
- [x] Accessibility and general questions are `h3`s

To make the contact section more discoverable, we could:
- [ ] Place the contact section _above_ the FAQ questions in the DOM
- [ ] Categorize the list of FAQ questions into subgroups with their own `h2`s to limit the number of `h2`s in total which in turn will make the contact `h2` more discoverable
